lib/imagec/download.go

```diff
@@ -104,7 +104,8 @@ func (s *serialProgressOutput) stop() {
 // It handles existing and simultaneous layer download de-duplication
 // This code is utilizes Docker's xfer package: https://github.com/docker/docker/tree/v1.11.2/distribution/xfer
 func (ldm *LayerDownloader) DownloadLayers(ctx context.Context, ic *ImageC) error {
```

Comment:
I'm inclined to either:
a. have the exported methods using plain `context.Context` and calling `trace.FromContext`, or
b. have all of the methods taking `trace.Operation`

At the moment we have a mix (see lib/imagec/docker.go)